src/main/java/com/revature/studyforce/flashcard/service/RatingService.java

```diff
@@ -1,33 +1,69 @@
 package com.revature.studyforce.flashcard.service;
 
+import com.revature.studyforce.flashcard.dto.RatingDTO;
+import com.revature.studyforce.flashcard.dto.RatingResponseDTO;
+import com.revature.studyforce.flashcard.model.Difficulty;
+import com.revature.studyforce.flashcard.model.Flashcard;
 import com.revature.studyforce.flashcard.model.Rating;
+import com.revature.studyforce.flashcard.repository.FlashcardRepository;
 import com.revature.studyforce.flashcard.repository.RatingRepository;
+import com.revature.studyforce.user.model.User;
+import com.revature.studyforce.user.repository.UserRepository;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+import java.util.List;
+import java.util.Optional;
 
 /**
  * Services for the Rating repository {@link RatingRepository}
  * @author Edson Rodriguez
  */
 @Service
 public class RatingService {
-    private final RatingRepository RATING_REPO;
+    private final RatingRepository ratingRepository;
+    private final FlashcardRepository flashcardRepository;
+    private final UserRepository userRepository;
 
     /**
-     * Constructor used to instantiate the repository
-     * @param ratingRepository  The repository used throughout the RatingService
+     * Constructor used to instantiate the necessary repository's
+     * @param ratingRepository {@link RatingRepository}
+     * @param flashcardRepo {@link FlashcardRepository}
+     * @param userRepository {@link UserRepository}
      */
     @Autowired
-    public RatingService(RatingRepository ratingRepository){
-        this.RATING_REPO = ratingRepository;
+    public RatingService(RatingRepository ratingRepository,FlashcardRepository flashcardRepo,UserRepository userRepository){
+        this.ratingRepository = ratingRepository;
+        this.flashcardRepository = flashcardRepo;
+        this.userRepository =userRepository;
     }
 
     /**
-     * Method to save a new rating
+     * Method used to store a rating
+     * @param ratingDTO The data transfer object that represents an incoming request to store a rating
+     * @return A data transfer object with the new rating score and the amount of ratings for that flashcard
      */
-    //TODO: change the return type
-    public Rating createRating(Rating rating){
-        return RATING_REPO.save(rating);
+    public RatingResponseDTO createRating(RatingDTO ratingDTO){
```

Comment:
theres too many documentation issues yo just the rest of this project needs a once over and correcting all docs